cedar-policy/src/lib.rs

```diff
@@ -42,5 +42,8 @@ pub use api::*;
 /// Frontend utilities, see comments in the module itself
 pub mod frontend;
 
+#[cfg(test)] //redundant, but helps rust_analyzer not incorrectly flag `use` statements
```

Comment:
I think this could also be `#![cfg(test)]` at the top of the test file. A little cleaner IMO.